node/network/collator-protocol/src/lib.rs

```diff
@@ -164,3 +164,129 @@ where
 		NetworkBridgeMessage::ReportPeer(peer, rep),
 	)).await;
 }
+
+pub(crate) mod fitness_metrics {
+    use super::{CollatorId, CollatorProtocolMessage, SubsystemContext, modify_reputation, PeerId};
```

Comment:
Please use tabs everywhere ;)

Reply:
Yes, I'm working on this PR right now. I will adjust everything once it's ready.